topics:
  input_map: "/map"

rrt:
  d: 0.2
  x_range: [-2, 2]
  y_range: [-2, 2]
  sec: 0.25
  it: 100

path:
  start: [0.0, 0.0]
  end: [0.0, 0.0]